tgui/packages/tgui/interfaces/Vote.js

```diff
@@ -0,0 +1,211 @@
+import { useBackend } from "../backend";
+import { Box, Icon, Stack, Button, Section, NoticeBox, LabeledList, Collapsible } from "../components";
+import { Window } from "../layouts";
+
+export const Vote = (props, context) => {
+  const { data } = useBackend(context);
+  const { mode, question, lower_admin } = data;
+
+  // Adds the voting type to title if there is an ongoing vote
+  let windowTitle = "Vote";
+  if (mode) {
+    windowTitle += ": " + (question || mode).replace(/^\w/, c => c.toUpperCase());
+  }
+
+  return (
+    <Window resizable title={windowTitle} width={400} height={500}>
```

Comment:
```suggestion
    <Window title={windowTitle} width={400} height={500}>
```

Reply:
Why don't you like resizing?

Reply:
All windows are resizable now regardless of the property